client/src/definitions/datasets.d.ts

```diff
@@ -7,29 +7,34 @@ type DataFormat =
   | "website"
   | "other";
 
+type UpdateFrequency =
+  | "never"
+  | "realtime"
+  | "daily"
+  | "weekly"
+  | "monthly"
+  | "yearly";
+
 export interface DatasetHeadlines {
   title: string;
   description: string;
 }
 
-export interface Dataset {
+export type Dataset = {
   id: string;
   createdAt: Date;
-  title: string;
-  description: string;
-  formats: DataFormat[];
-  entrypointEmail: string;
-  contactEmails: string[];
   headlines?: DatasetHeadlines;
-}
-
-export interface DatasetFormData {
   title: string;
   description: string;
   formats: DataFormat[];
   entrypointEmail: string;
   contactEmails: string[];
-}
+  service: string;
+  updateFrequency: string;
```

Comment:
Ça devrait être typé avec l'enum, non ?

```suggestion
  updateFrequency: UpdateFrequency;
```

Reply:
Ce sera corrigé par https://github.com/etalab/catalogage-donnees/pull/176